new_project/approval_template.md

```diff
@@ -0,0 +1,23 @@
+<!---
+Please ensure the [pre-evaluation requirements](github.com/niosh-mining/about/new_project/readme.md)
```

Comment:
Do we want to include those items here for them to check off?

Reply:
They should have checked them off before opening the issue but I guess it doesn't hurt to include them.